### PR TITLE
Add placeholder config for static placeholders

### PR DESCRIPTION
**Pull request by FinalAngel**
There is no need for releases in-between. There were no static placeholders before.

### CODE REVIEW

File: aldryn_jobs/cms_appconfig.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+from aldryn_apphooks_config.models import AppHookConfig
+from cms.models.fields import PlaceholderField
+
+
+class JobsConfig(AppHookConfig):
+    # Job PHFs
+    placeholder_jobs_top = PlaceholderField(
+        'jobs_top', related_name='aldryn_jobs_top')
+
```
Comment (intellectronica) [inline aldryn_jobs/cms_appconfig.py]: What's the point of adding a newline between every line?

**Comment by intellectronica** [comment]
:+1:

File: aldryn_jobs/cms_appconfig.py (hunk 1 above)
Comment (FinalAngel) [inline aldryn_jobs/cms_appconfig.py]: thats the style which is also used within aldryn-events and aldryn-newsblog

Comment (intellectronica) [inline aldryn_jobs/cms_appconfig.py]: Fair enough. I think it's unnecessary, but being consistent is important.

Comment (mkoistinen) [inline aldryn_jobs/cms_appconfig.py]: @intellectronica to be clear, it is customary to divide the imports into "sections" for clarity, separating python imports from Django imports from 3rd party imports from "local" imports.

Comment (intellectronica) [inline aldryn_jobs/cms_appconfig.py]: I wasn't talking about the imports but about the class attributes. Anyway, was just curious.

**Comment by mkoistinen** [comment]
I'm not 100% sure, but I think because we've moved the app_config to another file, the tests will need to be updated to reflect that.

**Comment by FinalAngel** [comment]
@cyberstar still needs to add the AppHook reload migration, so this probably will stay open for a bit :(

File: aldryn_jobs/migrations/0012_auto_20150713_0859.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import cms.models.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cms', '0012_auto_20150607_2207'),
```
Comment (cyberstar) [inline aldryn_jobs/migrations/0012_auto_20150713_0859.py]: will break cms 3.0.x versions.

PS. this note is a reminder.

Comment (FinalAngel) [inline aldryn_jobs/migrations/0012_auto_20150713_0859.py]: @cyberstar yes I fixed this actually within the other migration before rebase: https://github.com/aldryn/aldryn-jobs/commit/76f1210f0f0b6221ccf6dabae17a808719ac42be

**Comment by cyberstar** [comment]
rebased

**Comment by FinalAngel** [comment]
LGTM

File: aldryn_jobs/boilerplates/bootstrap3/templates/aldryn_jobs/jobs_detail.html [hunk 3]
```diff
@@ -2,11 +2,11 @@
 {% load i18n cms_tags apphooks_config_tags %}
 
 {% block jobs_content %}
-    {% static_placeholder "jobs_detail_top" %}
```
Comment (mkoistinen) [inline aldryn_jobs/boilerplates/bootstrap3/templates/aldryn_jobs/jobs_detail.html]: We need at least one "release" that have both placeholders present!

File: aldryn_jobs/boilerplates/bootstrap3/templates/aldryn_jobs/jobs_detail.html [hunk 4]
```diff
@@ -2,11 +2,11 @@
...
+    {% render_placeholder view.config.placeholder_jobs_detail_top %}
     <div class="aldryn-jobs-detail">
         {% include "aldryn_jobs/includes/job.html" with job_offer=object detail_view="true" %}
     </div>
-    {% static_placeholder "jobs_detail_top" %}
+    {% render_placeholder view.config.placeholder_jobs_detail_bottom %}
```
Comment (mkoistinen) [inline aldryn_jobs/boilerplates/bootstrap3/templates/aldryn_jobs/jobs_detail.html]: We need at least one "release" that have both placeholders present!

File: aldryn_jobs/boilerplates/bootstrap3/templates/aldryn_jobs/jobs_detail.html [hunk 5]
```diff
@@ -21,5 +21,5 @@
             </li>
         </ul>
     </div>
-    {% static_placeholder "jobs_detail_footer" %}
+    {% render_placeholder view.config.placeholder_jobs_detail_footer %}
```
Comment (mkoistinen) [inline aldryn_jobs/boilerplates/bootstrap3/templates/aldryn_jobs/jobs_detail.html]: We need at least one "release" that have both placeholders present!

**Comment by mkoistinen** [comment]
Looks great @cyberstar and @FinalAngel!

**Pull request merged**